Expecto.Tests/FocusedTests.fs

```diff
@@ -89,7 +89,10 @@ let all =
         ]
 
       // check if we can fail on focused tests
-      if runTests { defaultConfig with failOnFocusedTests = true } localList <> 1 then
+      if runTests { defaultConfig with
+                      failOnFocusedTests = true
+                      printer = TestPrinters.silent
+                      verbosity = Logging.LogLevel.Fatal } localList <> 1 then
```

Comment:
Let's do a newline here so that a cursory reader sees the comparison on the same column as `runTests`.